diff --git a/Libraries/Components/ScrollView/ScrollView.d.ts b/Libraries/Components/ScrollView/ScrollView.d.ts
index 2e26db757500d631d5ca503f58aa373f5032d4c2..52ff19ad1050c4183ffcd5d649db5c868d231637 100644
--- a/Libraries/Components/ScrollView/ScrollView.d.ts
+++ b/Libraries/Components/ScrollView/ScrollView.d.ts
@@ -19,6 +19,7 @@ import {
 import {RefreshControlProps} from '../RefreshControl/RefreshControl';
 import {Touchable} from '../Touchable/Touchable';
 import {ViewProps} from '../View/ViewPropTypes';
+import type { Props as ScrollViewStickyHeaderProps } from './ScrollViewStickyHeader';
 
 // See https://reactnative.dev/docs/scrollview#contentoffset
 export interface PointProp {
@@ -803,7 +804,7 @@ export interface ScrollViewProps
    * transforms, for example, when you want your list to have an animated and hidable header.
    * If component have not been provided, the default ScrollViewStickyHeader component will be used.
    */
-  StickyHeaderComponent?: React.ComponentType<any> | undefined;
+  StickyHeaderComponent?: React.ComponentType<ScrollViewStickyHeaderProps> | undefined;
 }
 
 declare class ScrollViewComponent extends React.Component<ScrollViewProps> {}
diff --git a/Libraries/Components/ScrollView/ScrollView.js b/Libraries/Components/ScrollView/ScrollView.js
index 39260c22dff9e263ea4252f011ea2409f33297f8..408b0e9dcb1a227d686942c39ba9caaeb965cbd3 100644
--- a/Libraries/Components/ScrollView/ScrollView.js
+++ b/Libraries/Components/ScrollView/ScrollView.js
@@ -1704,6 +1704,7 @@ class ScrollView extends React.Component<Props, State> {
           return (
             <StickyHeaderComponent
               key={key}
+              index={index}
               ref={ref => this._setStickyHeaderRef(key, ref)}
               nextHeaderLayoutY={this._headerLayoutYs.get(
                 this._getKeyForIndex(nextIndex, children),
diff --git a/Libraries/Components/ScrollView/ScrollViewStickyHeader.js b/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
index 052f01eb164d0e6e92a222986118c4ac52bb209a..316a738da16ae37cabda9584a347852ef0ff923e 100644
--- a/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
+++ b/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
@@ -30,6 +30,7 @@ export type Props = $ReadOnly<{
   scrollViewHeight: ?number,
   nativeID?: ?string,
   hiddenOnScroll?: ?boolean,
+  index: number;
 }>;
 
 type Instance = {
