diff --git a/Libraries/Components/ScrollView/ScrollView.d.ts b/Libraries/Components/ScrollView/ScrollView.d.ts
index b6e538ddc5a1be58b6227f461546fa6d84ca8871..c4cdff605fa44101de5f654e1680afb2c138054e 100644
--- a/Libraries/Components/ScrollView/ScrollView.d.ts
+++ b/Libraries/Components/ScrollView/ScrollView.d.ts
@@ -21,6 +21,7 @@ import {RefreshControlProps} from '../RefreshControl/RefreshControl';
 import {Touchable} from '../Touchable/Touchable';
 import {ViewProps} from '../View/ViewPropTypes';
 import {View} from '../View/View';
+import type { Props as ScrollViewStickyHeaderProps } from './ScrollViewStickyHeader';
 
 // See https://reactnative.dev/docs/scrollview#contentoffset
 export interface PointProp {
@@ -824,7 +825,7 @@ export interface ScrollViewProps
    * transforms, for example, when you want your list to have an animated and hidable header.
    * If component have not been provided, the default ScrollViewStickyHeader component will be used.
    */
-  StickyHeaderComponent?: React.ComponentType<any> | undefined;
+  StickyHeaderComponent?: React.ComponentType<ScrollViewStickyHeaderProps> | undefined;
 }
 
 declare class ScrollViewComponent extends React.Component<ScrollViewProps> {}
diff --git a/Libraries/Components/ScrollView/ScrollView.js b/Libraries/Components/ScrollView/ScrollView.js
index 8f4f297ed1520306f9b722af6c74d58a3c247aa0..18912e354c68a4b8eb67313f2a839545429ee125 100644
--- a/Libraries/Components/ScrollView/ScrollView.js
+++ b/Libraries/Components/ScrollView/ScrollView.js
@@ -1688,6 +1688,7 @@ class ScrollView extends React.Component<ScrollViewProps, ScrollViewState> {
           return (
             <StickyHeaderComponent
               key={key}
+              index={index}
               ref={ref => this._setStickyHeaderRef(key, ref)}
               nextHeaderLayoutY={this._headerLayoutYs.get(
                 this._getKeyForIndex(nextIndex, children),
diff --git a/Libraries/Components/ScrollView/ScrollViewStickyHeader.js b/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
index d1e5d6573f0933b567a5f14210d7aa12b52eb7b3..c96e151491c35167f7a94fd4d5b616b50a4c462f 100644
--- a/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
+++ b/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
@@ -37,6 +37,7 @@ export type ScrollViewStickyHeaderProps = $ReadOnly<{
   scrollViewHeight: ?number,
   nativeID?: ?string,
   hiddenOnScroll?: ?boolean,
+  index: number,
 }>;
 
 interface Instance extends React.ElementRef<typeof Animated.View> {
