diff --git a/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m b/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m
index bc6bb3d..d46b1ec 100644
--- a/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m
+++ b/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m
@@ -171,9 +171,10 @@ RCT_EXPORT_METHOD(onFinishRemoteNotification:(NSString *)notificationId fetchRes
 {
   if (self.remoteNotificationCallbacks) {
     RNCRemoteNotificationCallback completionHandler = [self.remoteNotificationCallbacks objectForKey:notificationId];
-    if (completionHandler) {
-      completionHandler(result);
-    }
+    // Edifice Badge fix: the lines below was causing a crash when receiving a foreground notification.
+    // if (completionHandler) {
+    //   completionHandler(result);
+    // }
     [self.remoteNotificationCallbacks removeObjectForKey:notificationId];
   }
 }
