diff --git a/node_modules/@react-native-firebase/messaging/ios/RNFBMessaging/RNFBMessaging+AppDelegate.m b/node_modules/@react-native-firebase/messaging/ios/RNFBMessaging/RNFBMessaging+AppDelegate.m
index 195579f..9957119 100644
--- a/node_modules/@react-native-firebase/messaging/ios/RNFBMessaging/RNFBMessaging+AppDelegate.m
+++ b/node_modules/@react-native-firebase/messaging/ios/RNFBMessaging/RNFBMessaging+AppDelegate.m
@@ -164,7 +164,8 @@
       // user JS code complete
       dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(25 * NSEC_PER_SEC)),
                      dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
-                       completionHandler(UIBackgroundFetchResultNewData);
+                       // Edifice Badge fix: the line below was causing a crash when receiving a background notification for the second time.
+                       // completionHandler(UIBackgroundFetchResultNewData);
 
                        // Stop background task after the longest timeout, async queue is okay to
                        // freeze again after handling period
