diff --git a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
index fbe863d..47f7686 100755
--- a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
+++ b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
@@ -7,16 +7,21 @@ import { TextField } from "react-native-material-textfield";
 const PasswordInputText = ({
   iconSize,
   iconColor,
+  invertVisibilityIcon,
   label,
   style,
   getRef,
   ...rest
 }) => {
-  const [eyeIcon, setEyeIcon] = useState("visibility-off");
+  const getVisibilityIcon = (isPassword) => isPassword ? 
+      ( invertVisibilityIcon ? "visibility-off" : "visibility") :
+      ( invertVisibilityIcon ? "visibility" : "visibility-off") ;
+
+  const [eyeIcon, setEyeIcon] = useState(getVisibilityIcon(false));
   const [isPassword, setIsPassword] = useState(true);
 
   const changePwdType = () => {
-    setEyeIcon(isPassword ? "visibility" : "visibility-off");
+    setEyeIcon(getVisibilityIcon(isPassword));
     setIsPassword((prevState) => !prevState);
   };
 
@@ -53,12 +58,14 @@ const styles = StyleSheet.create({
 
 PasswordInputText.defaultProps = {
   iconSize: 25,
+  invertVisibilityIcon: false,
   label: "Password",
   iconColor: "#222222",
 };
 
 PasswordInputText.propTypes = {
   iconSize: PropTypes.number,
+  invertVisibilityIcon: PropTypes.bool,
   label: PropTypes.string,
   iconColor: PropTypes.string,
 };
