diff --git a/node_modules/react-native-hide-show-password-input/package.json b/node_modules/react-native-hide-show-password-input/package.json
index 1c76cdd..6677769 100755
--- a/node_modules/react-native-hide-show-password-input/package.json
+++ b/node_modules/react-native-hide-show-password-input/package.json
@@ -14,7 +14,7 @@
   },
   "homepage": "https://github.com/vishaljadav24/react-native-hide-show-password-input#readme",
   "dependencies": {
-    "react-native-material-textfield": "vishaljadav24/react-native-material-textfield",
+    "rn-material-ui-textfield": "^1.0.7",
     "react-native-vector-icons": "^7.1.0",
     "prop-types": "^15.7.2"
   },
diff --git a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
index fbe863d..effe17f 100755
--- a/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
+++ b/node_modules/react-native-hide-show-password-input/src/component/passwordInput.js
@@ -2,21 +2,26 @@ import React, { useState } from "react";
 import { View, StyleSheet } from "react-native";
 import Icon from "react-native-vector-icons/MaterialIcons";
 import PropTypes from "prop-types";
-import { TextField } from "react-native-material-textfield";
+import { TextField } from "rn-material-ui-textfield";
 
 const PasswordInputText = ({
   iconSize,
   iconColor,
+  invertVisibilityIcon,
   label,
   style,
   getRef,
   ...rest
 }) => {
-  const [eyeIcon, setEyeIcon] = useState("visibility-off");
+  const getVisibilityIcon = (isPassword) => isPassword ? 
+      ( invertVisibilityIcon ? "visibility-off" : "visibility") :
+      ( invertVisibilityIcon ? "visibility" : "visibility-off") ;
+
+  const [eyeIcon, setEyeIcon] = useState(getVisibilityIcon(false));
   const [isPassword, setIsPassword] = useState(true);
 
   const changePwdType = () => {
-    setEyeIcon(isPassword ? "visibility" : "visibility-off");
+    setEyeIcon(getVisibilityIcon(isPassword));
     setIsPassword((prevState) => !prevState);
   };
 
@@ -53,12 +58,14 @@ const styles = StyleSheet.create({
 
 PasswordInputText.defaultProps = {
   iconSize: 25,
+  invertVisibilityIcon: false,
   label: "Password",
   iconColor: "#222222",
 };
 
 PasswordInputText.propTypes = {
   iconSize: PropTypes.number,
+  invertVisibilityIcon: PropTypes.bool,
   label: PropTypes.string,
   iconColor: PropTypes.string,
 };
