diff --git a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.d.ts b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.d.ts
index b6e538d..8ef8727 100644
--- a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.d.ts
+++ b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.d.ts
@@ -8,19 +8,21 @@
  */
 
 import type * as React from 'react';
-import {Constructor} from '../../../types/private/Utilities';
-import {Insets} from '../../../types/public/Insets';
-import {HostInstance} from '../../../types/public/ReactNativeTypes';
-import {ColorValue, StyleProp} from '../../StyleSheet/StyleSheet';
-import {ViewStyle} from '../../StyleSheet/StyleSheetTypes';
+import { Constructor } from '../../../types/private/Utilities';
+import { Insets } from '../../../types/public/Insets';
+import { HostInstance } from '../../../types/public/ReactNativeTypes';
+import { ColorValue, StyleProp } from '../../StyleSheet/StyleSheet';
+import { ViewStyle } from '../../StyleSheet/StyleSheetTypes';
 import {
   NativeSyntheticEvent,
   NativeTouchEvent,
 } from '../../Types/CoreEventTypes';
-import {RefreshControlProps} from '../RefreshControl/RefreshControl';
-import {Touchable} from '../Touchable/Touchable';
-import {ViewProps} from '../View/ViewPropTypes';
-import {View} from '../View/View';
+import { RefreshControlProps } from '../RefreshControl/RefreshControl';
+import { Touchable } from '../Touchable/Touchable';
+import { View } from '../View/View';
+import { ViewProps } from '../View/ViewPropTypes';
+
+import type { Props as ScrollViewStickyHeaderProps } from './ScrollViewStickyHeader';
 
 // See https://reactnative.dev/docs/scrollview#contentoffset
 export interface PointProp {
@@ -824,7 +826,7 @@ export interface ScrollViewProps
    * transforms, for example, when you want your list to have an animated and hidable header.
    * If component have not been provided, the default ScrollViewStickyHeader component will be used.
    */
-  StickyHeaderComponent?: React.ComponentType<any> | undefined;
+  StickyHeaderComponent?: React.ComponentType<ScrollViewStickyHeaderProps> | undefined;
 }
 
 declare class ScrollViewComponent extends React.Component<ScrollViewProps> {}
diff --git a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
index 8f4f297..18912e3 100644
--- a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
+++ b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
@@ -1688,6 +1688,7 @@ class ScrollView extends React.Component<ScrollViewProps, ScrollViewState> {
           return (
             <StickyHeaderComponent
               key={key}
+              index={index}
               ref={ref => this._setStickyHeaderRef(key, ref)}
               nextHeaderLayoutY={this._headerLayoutYs.get(
                 this._getKeyForIndex(nextIndex, children),
diff --git a/node_modules/react-native/Libraries/Components/ScrollView/ScrollViewStickyHeader.js b/node_modules/react-native/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
index d1e5d65..c96e151 100644
--- a/node_modules/react-native/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
+++ b/node_modules/react-native/Libraries/Components/ScrollView/ScrollViewStickyHeader.js
@@ -37,6 +37,7 @@ export type ScrollViewStickyHeaderProps = $ReadOnly<{
   scrollViewHeight: ?number,
   nativeID?: ?string,
   hiddenOnScroll?: ?boolean,
+  index: number,
 }>;
 
 interface Instance extends React.ElementRef<typeof Animated.View> {
