diff --git a/node_modules/react-native-tiny-toast/lib/ToastContainer.js b/node_modules/react-native-tiny-toast/lib/ToastContainer.js
index e95eb98..7e91dbc 100644
--- a/node_modules/react-native-tiny-toast/lib/ToastContainer.js
+++ b/node_modules/react-native-tiny-toast/lib/ToastContainer.js
@@ -1,7 +1,6 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import {
-  ViewPropTypes,
   StyleSheet,
   View,
   Text,
@@ -13,6 +12,8 @@ import {
 } from 'react-native'
 import {getStatusBarHeight, getBottomSpace} from 'react-native-iphone-x-helper'
 
+import { ImagePropTypes, TextPropTypes, ViewPropTypes } from 'deprecated-react-native-prop-types';
+
 const position = {
   TOP: 40 + (Platform.OS === 'ios' ? getStatusBarHeight() : 0),
   BOTTOM: -40 - (Platform.OS === 'ios' ? getBottomSpace() : 0),
@@ -38,12 +39,12 @@ class ToastContainer extends Component {
     shadowColor: PropTypes.string,
     showText: PropTypes.bool,
     textColor: PropTypes.string,
-    textStyle: Text.propTypes.style,
+    textStyle: TextPropTypes.style,
     mask: PropTypes.bool,
     maskColor: PropTypes.string,
     maskStyle: ViewPropTypes.style,
     imgSource: PropTypes.any,
-    imgStyle: Image.propTypes.style,
+    imgStyle: ImagePropTypes.style,
     loading: PropTypes.bool,
     indicatorSize: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     onHidden: PropTypes.func,
