diff --git a/node_modules/react-native-rate/android/.gradle/9.2.0/checksums/checksums.lock b/node_modules/react-native-rate/android/.gradle/9.2.0/checksums/checksums.lock
new file mode 100644
index 0000000..82b3122
Binary files /dev/null and b/node_modules/react-native-rate/android/.gradle/9.2.0/checksums/checksums.lock differ
diff --git a/node_modules/react-native-rate/android/.gradle/9.2.0/fileChanges/last-build.bin b/node_modules/react-native-rate/android/.gradle/9.2.0/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-rate/android/.gradle/9.2.0/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.bin b/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.bin
new file mode 100644
index 0000000..dc059cf
Binary files /dev/null and b/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.bin differ
diff --git a/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.lock b/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..23444a6
Binary files /dev/null and b/node_modules/react-native-rate/android/.gradle/9.2.0/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-rate/android/.gradle/9.2.0/gc.properties b/node_modules/react-native-rate/android/.gradle/9.2.0/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..04b073f
Binary files /dev/null and b/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..1800071
--- /dev/null
+++ b/node_modules/react-native-rate/android/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Wed Aug 26 09:00:47 CEST 2026
+gradle.version=9.2.0
diff --git a/node_modules/react-native-rate/android/.gradle/vcs-1/gc.properties b/node_modules/react-native-rate/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-rate/ios/RNRate.m b/node_modules/react-native-rate/ios/RNRate.m
index d1e6cb4..15d4f15 100644
--- a/node_modules/react-native-rate/ios/RNRate.m
+++ b/node_modules/react-native-rate/ios/RNRate.m
@@ -1,6 +1,6 @@
 #import "RNRate.h"
-#import "RCTBridgeModule.h"
-#import "React/RCTConvert.h"
+#import <React/RCTBridgeModule.h>
+#import <React/RCTConvert.h>
 
 @implementation RNRate
 RCT_EXPORT_MODULE();
@@ -88,11 +88,14 @@ RCT_EXPORT_METHOD(rate: (NSDictionary *)options : (RCTResponseSenderBlock) callb
 }
 
 - (void) openAppStoreAndRate : (NSString *) url {
-  dispatch_async(dispatch_get_main_queue(), ^{
-    [[UIApplication sharedApplication] openURL: [NSURL URLWithString:url]];
-  });
+    dispatch_async(dispatch_get_main_queue(), ^{
+        NSURL *nsurl = [NSURL URLWithString:url];
+        if (@available(iOS 10.0, *)) {
+            [[UIApplication sharedApplication] openURL:nsurl options:@{} completionHandler:nil];
+        } else {
+            [[UIApplication sharedApplication] openURL:nsurl];
+        }
+    });
 }
 
-
-
 @end
