diff --git a/node_modules/react-native-swipeable/lib/index.js b/node_modules/react-native-swipeable/lib/index.js
index 57996ad..2fa8d4e 100644
--- a/node_modules/react-native-swipeable/lib/index.js
+++ b/node_modules/react-native-swipeable/lib/index.js
@@ -16,6 +16,8 @@ var _reactNative = require('react-native');
 
 var _propTypes = require('prop-types');
 
+var _deprecated = require('deprecated-react-native-prop-types');
+
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }
@@ -603,12 +605,12 @@ Swipeable.propTypes = {
   swipeStartMinDistance: _propTypes.PropTypes.number,
 
   // styles
-  style: _reactNative.ViewPropTypes.style,
-  leftContainerStyle: _reactNative.ViewPropTypes.style,
-  leftButtonContainerStyle: _reactNative.ViewPropTypes.style,
-  rightContainerStyle: _reactNative.ViewPropTypes.style,
-  rightButtonContainerStyle: _reactNative.ViewPropTypes.style,
-  contentContainerStyle: _reactNative.ViewPropTypes.style
+  style: _deprecated.ViewPropTypes.style_reactNative,
+  leftContainerStyle: _deprecated.ViewPropTypes.style,
+  leftButtonContainerStyle: _deprecated.ViewPropTypes.style,
+  rightContainerStyle: _deprecated.ViewPropTypes.style,
+  rightButtonContainerStyle: _deprecated.ViewPropTypes.style,
+  contentContainerStyle: _deprecated.ViewPropTypes.style
 };
 Swipeable.defaultProps = {
   leftContent: null,
