diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
index a0f633f..c95fab8 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
@@ -173,7 +173,8 @@ public class ImagePickerModule extends ReactContextBaseJavaModule implements Act
     public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
 
         // onActivityResult is called even when ActivityNotFoundException occurs
-        if (!isValidRequestCode(requestCode) || (this.callback == null)) {
+        if (!isValidRequestCode(requestCode) || (this.callback == null) ||
+           ((requestCode == REQUEST_LAUNCH_LIBRARY) && (data == null))) {
             return;
         }
 
