diff --git a/node_modules/react-native-dropdown-picker/index.d.ts b/node_modules/react-native-dropdown-picker/index.d.ts
index 9848ccc..8f624de 100644
--- a/node_modules/react-native-dropdown-picker/index.d.ts
+++ b/node_modules/react-native-dropdown-picker/index.d.ts
@@ -221,6 +221,9 @@ declare module 'react-native-dropdown-picker' {
     showTickIcon?: boolean;
     stickyHeader?: boolean;
     autoScroll?: boolean;
+    BodyComponent?: (props: {
+      style: StyleProp<ViewStyle>;
+    }) => JSX.Element;
     ArrowUpIconComponent?: (props: {
       style: StyleProp<ViewStyle>;
     }) => JSX.Element;
diff --git a/node_modules/react-native-dropdown-picker/package.json b/node_modules/react-native-dropdown-picker/package.json
index a3f8cba..e85d698 100644
--- a/node_modules/react-native-dropdown-picker/package.json
+++ b/node_modules/react-native-dropdown-picker/package.json
@@ -43,9 +43,6 @@
   "author": "Hossein Zare",
   "main": "index.js",
   "types": "index.d.ts",
-  "dependencies": {
-    "react-native-gesture-handler": "*"
-  },
   "peerDependencies": {
     "react": "*",
     "react-native": "*"
diff --git a/node_modules/react-native-dropdown-picker/src/components/Picker.js b/node_modules/react-native-dropdown-picker/src/components/Picker.js
index 08c6a68..07f96ac 100644
--- a/node_modules/react-native-dropdown-picker/src/components/Picker.js
+++ b/node_modules/react-native-dropdown-picker/src/components/Picker.js
@@ -41,7 +41,8 @@ import {
     GET_DROPDOWN_DIRECTION,
     LANGUAGE,
     RTL_DIRECTION,
-    RTL_STYLE
+    RTL_STYLE,
+    ARROW_ICON_POSITION,
 } from '../constants';
 import THEMES from '../themes';
 import RenderBadgeItem from './RenderBadgeItem';
@@ -121,6 +122,7 @@ function Picker({
     showTickIcon = true,
     stickyHeader = false,
     autoScroll = false,
+    BodyComponent = null,
     ArrowUpIconComponent = null,
     ArrowDownIconComponent = null,
     TickIconComponent = null,
@@ -162,6 +164,7 @@ function Picker({
     testID,
     closeOnBackPressed = false,
     extendableBadgeContainer = false,
+    arrowIconPosition = ARROW_ICON_POSITION.RIGHT,
     onSelectItem = (item) => {}
 }) {
     const [necessaryItems, setNecessaryItems] = useState([]);
@@ -902,14 +905,16 @@ function Picker({
      * The simple body component.
      * @returns {JSX.Element}
      */
-    const SimpleBodyComponent = useMemo(() => (
+    const SimpleBodyComponent =  useMemo(() => (
         <>
             {SelectedItemIconComponent}
-            <Text style={_labelStyle} {...labelProps}>
-                {_selectedItemLabel}
-            </Text>
+            {BodyComponent !== null ? <BodyComponent /> : (
+                <Text style={_labelStyle} {...labelProps}>
+                    {_selectedItemLabel}
+                </Text>
+            )}
         </>
-    ), [SelectedItemIconComponent, _labelStyle, labelProps, _selectedItemLabel]);
+    ), [SelectedItemIconComponent, BodyComponent, _labelStyle, labelProps, _selectedItemLabel]);
 
     /**
      * onPress badge.
@@ -1808,14 +1813,19 @@ function Picker({
      */
     const pointerEvents = useMemo(() => disabled ? "none" : "auto", [disabled]);
 
+    const TopComponent = useMemo(() => {
+        switch (arrowIconPosition) {
+            case ARROW_ICON_POSITION.RIGHT: return [_BodyComponent, _ArrowComponent];
+            default: return [_ArrowComponent, _BodyComponent];
+        }
+    }, [arrowIconPosition, _ArrowComponent, _BodyComponent]);
+
     return (
         <>
             <View style={_containerStyle} {...containerProps}>
                 <TouchableOpacity style={_style} onPress={__onPress} onLayout={__onLayout} {...props} ref={onRef} pointerEvents={pointerEvents} disabled={disabled} testID={testID}>
-                    {_BodyComponent}
-                    {_ArrowComponent}
+                    {TopComponent}
                 </TouchableOpacity>
-
                 {DropDownBodyComponent}
             </View>
         </>
diff --git a/node_modules/react-native-dropdown-picker/src/constants/index.js b/node_modules/react-native-dropdown-picker/src/constants/index.js
index 23d34eb..2bbfe60 100644
--- a/node_modules/react-native-dropdown-picker/src/constants/index.js
+++ b/node_modules/react-native-dropdown-picker/src/constants/index.js
@@ -134,3 +134,5 @@ export const ASCII_CODE = (str) => {
 
     return chr;
 }
+
+export const ARROW_ICON_POSITION = { LEFT: 'LEFT', RIGHT: 'RIGHT' };
