diff --git a/node_modules/react-native-matomo/ios/MatomoTracker/MatomoTracker.swift b/node_modules/react-native-matomo/ios/MatomoTracker/MatomoTracker.swift
index 1b4fca8..baaa2c0 100755
--- a/node_modules/react-native-matomo/ios/MatomoTracker/MatomoTracker.swift
+++ b/node_modules/react-native-matomo/ios/MatomoTracker/MatomoTracker.swift
@@ -115,7 +115,7 @@ final public class MatomoTracker: NSObject {
     ///   - baseURL: The url of the Matomo server. This url has to end in `piwik.php`.
     ///   - userAgent: An optional parameter for custom user agent.
     @objc convenience public init(siteId: String, baseURL: URL, userAgent: String? = nil) {
-        assert(baseURL.absoluteString.hasSuffix("piwik.php"), "The baseURL is expected to end in piwik.php")
+        //assert(baseURL.absoluteString.hasSuffix("piwik.php"), "The baseURL is expected to end in piwik.php")
         
         let queue = MemoryQueue()
         let dispatcher = URLSessionDispatcher(baseURL: baseURL, userAgent: userAgent)
