diff --git a/node_modules/react-native-media-console/lib/typescript/types.d.ts b/node_modules/react-native-media-console/lib/typescript/types.d.ts
index baa4193..15e6cb9 100644
--- a/node_modules/react-native-media-console/lib/typescript/types.d.ts
+++ b/node_modules/react-native-media-console/lib/typescript/types.d.ts
@@ -1,8 +1,8 @@
-import type { RefObject } from 'react';
-import type { ViewStyle, StyleProp, Animated } from 'react-native';
+import type { ReactElement, RefObject } from 'react';
+import type { Animated, StyleProp, ViewStyle } from 'react-native';
 import type Reanimated from 'react-native-reanimated';
 import type { StyleProps } from 'react-native-reanimated';
-import type { VideoRef, ReactVideoProps } from 'react-native-video';
+import type { ReactVideoProps, VideoRef } from 'react-native-video';
 export declare type WithRequiredProperty<Type, Key extends keyof Type> = Type & {
     [Property in Key]-?: Type[Property];
 };
@@ -230,4 +230,9 @@ export interface VideoPlayerProps extends ReactVideoProps {
      * testID selector for testing
      */
     testID?: string;
+    /**
+     * Element to replace video
+     *
+    */
+    posterElement?: ReactElement;
 }
diff --git a/node_modules/react-native-media-console/src/VideoPlayer.tsx b/node_modules/react-native-media-console/src/VideoPlayer.tsx
index ada0347..a73fdba 100644
--- a/node_modules/react-native-media-console/src/VideoPlayer.tsx
+++ b/node_modules/react-native-media-console/src/VideoPlayer.tsx
@@ -1,5 +1,5 @@
-import React, {useCallback, useState, useEffect, useRef} from 'react';
-import {View} from 'react-native';
+import React, { useCallback, useEffect, useImperativeHandle, useRef, useState } from 'react';
+import { View } from 'react-native';
 import Video, {
   OnLoadData,
   OnLoadStartData,
@@ -8,19 +8,19 @@ import Video, {
   ResizeMode,
   VideoRef,
 } from 'react-native-video';
-import {useControlTimeout, useJSAnimations, usePanResponders} from './hooks';
 import {
+  BottomControls,
   Error,
   Loader,
-  TopControls,
-  BottomControls,
-  PlayPause,
   Overlay,
+  PlayPause,
+  TopControls,
 } from './components';
-import {PlatformSupport} from './OSSupport';
-import {_onBack} from './utils';
-import {_styles} from './styles';
-import type {VideoPlayerProps, WithRequiredProperty} from './types';
+import { useControlTimeout, useJSAnimations, usePanResponders } from './hooks';
+import { PlatformSupport } from './OSSupport';
+import { _styles } from './styles';
+import type { VideoPlayerProps, WithRequiredProperty } from './types';
+import { _onBack } from './utils';
 
 const volumeWidth = 150;
 const iconOffset = 0;
@@ -76,6 +76,7 @@ const AnimatedVideoPlayer = (
     rewindTime = 15,
     pan: {horizontal: horizontalPan, inverted: invertedPan} = {},
     testID,
+    posterElement,
   } = props;
 
   const mounted = useRef(false);
@@ -107,8 +108,9 @@ const AnimatedVideoPlayer = (
   const [currentTime, setCurrentTime] = useState(0);
   const [error, setError] = useState(false);
   const [duration, setDuration] = useState(0);
+  const [_isEnd, setIsEnd] = useState<boolean>(false);
 
-  const videoRef = props.videoRef || _videoRef;
+const internalVideoRef = _videoRef;
 
   const toggleFullscreen = () => setIsFullscreen((prevState) => !prevState);
   const toggleControls = () =>
@@ -118,16 +120,41 @@ const AnimatedVideoPlayer = (
     setPaused((prevState) => !prevState);
   };
 
+  useImperativeHandle(
+  props.videoRef,
+  () => ({
+    seek: (time: number, tolerance?: number) => internalVideoRef.current?.seek(time, tolerance),
+    resume: () => internalVideoRef.current?.resume(),
+    pause: () => internalVideoRef.current?.pause(),
+    presentFullscreenPlayer: () => internalVideoRef.current?.presentFullscreenPlayer(),
+    dismissFullscreenPlayer: () => internalVideoRef.current?.dismissFullscreenPlayer(),
+    restoreUserInterfaceForPictureInPictureStopCompleted: (restore: boolean) => internalVideoRef.current?.restoreUserInterfaceForPictureInPictureStopCompleted(restore),
+    save: (options: object) => internalVideoRef.current?.save(options),
+    setVolume: (volume: number) => internalVideoRef.current?.setVolume(volume),
+    getCurrentPosition: () => internalVideoRef.current?.getCurrentPosition() || Promise.resolve(0),
+    setFullScreen: (fullScreen: boolean) => internalVideoRef.current?.setFullScreen(fullScreen),
+    setSource: (source?: any) => internalVideoRef.current?.setSource(source),
+    enterPictureInPicture: () => internalVideoRef.current?.enterPictureInPicture(),
+    exitPictureInPicture: () => internalVideoRef.current?.exitPictureInPicture(),
+    toggleControls,
+  }),
+  [toggleControls],
+);
+
+const videoRef = props.videoRef || internalVideoRef;
+
   const styles = {
     videoStyle,
     containerStyle: containerStyle,
   };
 
   const _onSeek = (obj: OnSeekData) => {
-    if (!seeking) {
-      setControlTimeout();
+    setControlTimeout();
+    setCurrentTime(obj.currentTime);
+    if (_isEnd) {
+      setCurrentTime(duration);
+      setIsEnd(false)
     }
-    setCurrentTime(obj.seekTime);
 
     if (typeof onSeek === 'function') {
       onSeek(obj);
@@ -138,6 +165,7 @@ const AnimatedVideoPlayer = (
     if (currentTime < duration) {
       setCurrentTime(duration);
       setPaused(!props.repeat);
+      setIsEnd(true);
 
       if (showOnEnd) {
         setShowControls(!props.repeat);
@@ -404,15 +432,16 @@ const AnimatedVideoPlayer = (
         <Video
           {...props}
           {...events}
-          ref={videoRef || _videoRef}
+          ref={internalVideoRef}          
           resizeMode={_resizeMode}
           volume={_volume}
           paused={_paused}
           muted={_muted}
           rate={rate}
-          style={[_styles.player.video, styles.videoStyle]}
+          style={[_styles.player.video, styles.videoStyle, {...posterElement ? {opacity: 0}: {}}]}
           source={source}
         />
+        {posterElement ?? null}
         {loading ? (
           <Loader />
         ) : (
@@ -432,19 +461,32 @@ const AnimatedVideoPlayer = (
               showControls={showControls}
             />
             <PlayPause
-              animations={animations}
-              disablePlayPause={disablePlayPause}
-              disableSeekButtons={disableSeekButtons}
-              paused={_paused}
-              togglePlayPause={togglePlayPause}
-              resetControlTimeout={resetControlTimeout}
-              showControls={showControls}
-              onPressRewind={() =>
-                videoRef?.current?.seek(currentTime - rewindTime)
-              }
-              onPressForward={() =>
-                videoRef?.current?.seek(currentTime + rewindTime)
-              }
+                animations={animations}
+                disablePlayPause={disablePlayPause}
+                disableSeekButtons={disableSeekButtons}
+                paused={_paused}
+                togglePlayPause={togglePlayPause}
+                resetControlTimeout={resetControlTimeout}
+                showControls={showControls}
+                onPressRewind={() => {
+                  if (currentTime !== 0) {
+                    if (currentTime - rewindTime > 0) {
+                      videoRef?.current?.seek(currentTime - rewindTime)
+                    } else {
+                      setSeekerPosition(0)
+                      videoRef?.current?.seek(0)
+                    }
+                  }
+                }}
+                onPressForward={() => {
+                  if (Math.floor(currentTime) < Math.floor(duration)) {
+                    if (currentTime + rewindTime > duration) {
+                      videoRef?.current?.seek(duration - 0.01)
+                    } else {
+                      videoRef?.current?.seek(currentTime + rewindTime)
+                    }
+                  }
+                }}
             />
             <BottomControls
               animations={animations}
diff --git a/node_modules/react-native-media-console/src/components/Back/Back.tsx b/node_modules/react-native-media-console/src/components/Back/Back.tsx
index ef51f8d..e24c1bf 100644
--- a/node_modules/react-native-media-console/src/components/Back/Back.tsx
+++ b/node_modules/react-native-media-console/src/components/Back/Back.tsx
@@ -1,6 +1,6 @@
 import React from 'react';
-import {Image} from 'react-native';
-import {Control} from '../Control';
+import { Image } from 'react-native';
+import { Control } from '../Control';
 
 interface BackProps {
   onBack: () => void;
@@ -11,7 +11,7 @@ interface BackProps {
 export const Back = ({onBack, showControls}: BackProps) => {
   return (
     <Control callback={onBack} disabled={!showControls}>
-      <Image source={require('../../assets/img/back.png')} />
+      <Image source={require('ASSETS/icons/bitmapIcons/close.png')} />
     </Control>
   );
 };
diff --git a/node_modules/react-native-media-console/src/components/BottomControls.tsx b/node_modules/react-native-media-console/src/components/BottomControls.tsx
index 86a949d..34e87b7 100644
--- a/node_modules/react-native-media-console/src/components/BottomControls.tsx
+++ b/node_modules/react-native-media-console/src/components/BottomControls.tsx
@@ -1,18 +1,21 @@
-import React, {Dispatch, SetStateAction} from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
 import {
+  GestureResponderHandlers,
   ImageBackground,
+  Platform,
   SafeAreaView,
   StyleSheet,
-  GestureResponderHandlers,
+  View
 } from 'react-native';
-import {Timer} from './Timer';
-import {Title} from './Title';
-import {NullControl} from './NullControl';
-import {Fullscreen} from './Fullscreen';
-import {Seekbar} from './Seekbar';
-import {calculateTime} from '../utils';
-import type {VideoAnimations} from '../types';
-import {styles} from './styles';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import type { VideoAnimations } from '../types';
+import { calculateTime } from '../utils';
+import { Fullscreen } from './Fullscreen';
+import { NullControl } from './NullControl';
+import { Seekbar } from './Seekbar';
+import { styles } from './styles';
+import { Timer } from './Timer';
+import { Title } from './Title';
 
 interface BottomControlsProps {
   showControls: boolean;
@@ -60,6 +63,11 @@ export const BottomControls = ({
   disableFullscreen,
   toggleFullscreen,
 }: BottomControlsProps) => {
+  const insets = useSafeAreaInsets();
+  const dynamicBottomPadding = Platform.select({
+          android: Math.max(insets.bottom, 30),
+          ios: Math.max(insets.bottom * 0.3, 4),
+        });
   const timerControl = disableTimer ? (
     <NullControl />
   ) : (
@@ -116,9 +124,9 @@ export const BottomControls = ({
           <Title title={title} />
           {fullscreenControl}
         </SafeAreaView>
-        <SafeAreaView style={styles.seekBarContainer}>
+        <View style={[styles.seekBarContainer, {paddingBottom: dynamicBottomPadding}]}>
           {seekbarControl}
-        </SafeAreaView>
+        </View>
       </ImageBackground>
     </AnimatedView>
   );
diff --git a/node_modules/react-native-media-console/src/components/styles.ts b/node_modules/react-native-media-console/src/components/styles.ts
index 380b624..1823313 100644
--- a/node_modules/react-native-media-console/src/components/styles.ts
+++ b/node_modules/react-native-media-console/src/components/styles.ts
@@ -1,4 +1,4 @@
-import {StyleSheet} from 'react-native';
+import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
   row: {
@@ -26,5 +26,6 @@ export const styles = StyleSheet.create({
   },
   seekBarContainer: {
     width: '100%',
+    paddingVertical: 8,
   },
 });
diff --git a/node_modules/react-native-media-console/src/types.ts b/node_modules/react-native-media-console/src/types.ts
index 87c401b..781dd10 100644
--- a/node_modules/react-native-media-console/src/types.ts
+++ b/node_modules/react-native-media-console/src/types.ts
@@ -1,8 +1,8 @@
-import type {RefObject} from 'react';
-import type {ViewStyle, StyleProp, Animated} from 'react-native';
+import type { ReactElement, RefObject } from 'react';
+import type { Animated, StyleProp, ViewStyle } from 'react-native';
 import type Reanimated from 'react-native-reanimated';
-import type {StyleProps} from 'react-native-reanimated';
-import type {VideoRef, ReactVideoProps} from 'react-native-video';
+import type { StyleProps } from 'react-native-reanimated';
+import type { ReactVideoProps, VideoRef } from 'react-native-video';
 
 export type WithRequiredProperty<Type, Key extends keyof Type> = Type & {
   [Property in Key]-?: Type[Property];
@@ -268,4 +268,10 @@ export interface VideoPlayerProps extends ReactVideoProps {
    * testID selector for testing
    */
   testID?: string;
+
+  /**
+  * Element to replace video
+  *
+  */
+  posterElement?: ReactElement;
 }
