diff --git a/node_modules/react-native-shake/react-native-shake.podspec b/node_modules/react-native-shake/react-native-shake.podspec
index d4d1bb9..a72a37a 100644
--- a/node_modules/react-native-shake/react-native-shake.podspec
+++ b/node_modules/react-native-shake/react-native-shake.podspec
@@ -15,6 +15,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/Doko-Demo-Doa/react-native-shake.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm,cpp}"
+  s.exclude_files = "ios/generated/**/*"
 
   # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
   # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
