diff --git a/node_modules/rn-material-ui-textfield/src/components/field/index.js b/node_modules/rn-material-ui-textfield/src/components/field/index.js
index c5f1e84..ef8ba25 100644
--- a/node_modules/rn-material-ui-textfield/src/components/field/index.js
+++ b/node_modules/rn-material-ui-textfield/src/components/field/index.js
@@ -246,7 +246,7 @@ export default class TextField extends PureComponent {
   setNativeProps(props) {
     let { current: input } = this.inputRef;
 
-    input.setNativeProps(props);
+    input?.setNativeProps(props);
   }
 
   focusState() {
@@ -270,20 +270,20 @@ export default class TextField extends PureComponent {
     let { current: input } = this.inputRef;
 
     if (!disabled && editable) {
-      input.focus();
+      input?.focus();
     }
   }
 
   blur() {
     let { current: input } = this.inputRef;
 
-    input.blur();
+    input?.blur();
   }
 
   clear() {
     let { current: input } = this.inputRef;
 
-    input.clear();
+    input?.clear();
 
     /* onChangeText is not triggered by .clear() */
     this.onChangeText('');
@@ -308,7 +308,7 @@ export default class TextField extends PureComponent {
   isFocused() {
     let { current: input } = this.inputRef;
 
-    return input.isFocused();
+    return input?.isFocused();
   }
 
   isRestricted() {
